.github/ROADMAP.md

```diff
@@ -12,11 +12,10 @@ The team will release one minor version upgrade each quarter, or whenever a new
 
 
 2022 Q4:
-- 2D Semantic Segmentation task, the team focuses on DeepLabV3 and opens up contribution
-  for UNet/FCN.
-- 3D Object Detection task, the team focuses on providing point cloud data pipeline.
+- 2D Semantic Segmentation task, the team focuses on DeepLabV3, DeepLabV3+ and opens up contribution for UNet/FCN.
```

Comment:
Do we want to include VOS related archs in this point or it is too early?

Reply:
Are you referring VOS of [this](https://openreview.net/forum?id=TW7d65uYu5M)?

Reply:
@innat Or probably you are looking for VIT:
https://github.com/sukjunhwang/VITA
https://arxiv.org/abs/2211.08834

Reply:
> No, I was talking about VOS Challenge models ranked at: https://www.votchallenge.net/vot2022/
> 
> Like e.g. [google-research/scenic#587](https://github.com/google-research/scenic/issues/587)

what is the unique value of this, compared to our existing offerings?

Reply:
> what is the unique value of this, compared to our existing offerings?

Are you talking about scenic or VOS models?

Reply:
> No, I was talking about VOS Challenge models ranked at: https://www.votchallenge.net/vot2022/
> 
> Like e.g. [google-research/scenic#587](https://github.com/google-research/scenic/issues/587)

asking about this

Reply:
These are two different things:
Scenic is a CV library for JAX + Flax and that one is a related request for VOS models.

Keras-cv is TF based and the comment here is a sort of parallel request to explore if we are interested in VOS/VIS models for the roadmap.